Gulpfile.js

```diff
@@ -0,0 +1,55 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var postcss = require('gulp-postcss');
+var sourcemaps = require('gulp-sourcemaps');
+var autoprefixer = require('autoprefixer');
+var buffer = require('vinyl-buffer');
+var imagemin = require('gulp-imagemin');
+var merge = require('merge-stream');
+var spritesmith = require('gulp.spritesmith');
+var browserSync = require('browser-sync').create();
+
+// sass, postcss, autoprefixer task
+gulp.task('styles', function() {
+  gulp.src('sass/**/*.scss')
```

Comment:
Every gulp task should return. Here and below. And I told you what it could be :)